fsharp-backend/tests/testfiles/int.tests

```diff
@@ -12,6 +12,8 @@ Int.min_v0 50 -10 = -10
 Int.negate_v0 -5 = 5
 Int.negate_v0 5 = -5
 //Int.remainder_v0 5 0 = Error "`divisor` must be non-zero"
+Int.mod_v0 15 5 = 0
+Int.mod_v0 15 6 = 3
```

Comment:
Can you add a test for zero? Can you also add a test for 2 numbers, both larger than 64 bits?

Reply:
Also, these tests look right to me.

Reply:
These are some results for numbers larger than 64 bits
![image](https://user-images.githubusercontent.com/13671857/101271280-4d535280-3736-11eb-8b2c-54d11971c50d.png)
![image](https://user-images.githubusercontent.com/13671857/101271288-61974f80-3736-11eb-81c1-d4c6ac79d96f.png)

Reply:
Is this still the case after using BigInt.remainder? I'm imagining this happened because the numbers got truncated.

Reply:
Yes, it is. I did that test after changing to BigInt.remainder. I will try to find the root cause. Thank you!